packages/lit-dev-content/site/tutorials/content/pointer-events/02.md

```diff
@@ -0,0 +1,24 @@
+Pointer Events extend from Mouse Events, have access to all the same [properties](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent).
+
+In addition, Pointer Events have several new properties:
+
+  * `pointerType`
+    * This property will be one of `mouse`, `touch`, or `pen`, and allows you to determine the input source of the event.
+  * `isPrimary`
+    * This property will be `true` if the pointer firing events is considered the "primary" pointer for the interaction, and will generate `click` events and other UI interactions.
+    * For `pointerType === mouse`, this will always be `true`
+    * For `pen` or `touch` input, this will be true only for the first finger or pen to touch the screen.
+    * If you want to ignore multi-touch interactions, you can ignore pointers where `isPrimary === false`
+  * `pressure`
+    * This property is a number between `0` and `1`, determined by how much pressure was applied relative to the maximum pressure the device can measure
+    * When the device cannot determine pressure, this will read `0.5` when the pointer is down, and `0` otherwise.
+  * `height` and `width`
+    * These properties will give the CSS pixel size of the pointer.
+    * This will always be `1` for mouse.
+  * `tiltX` and `tiltY`
+    * These properties will report the angle of the pen to the screen when pressed in the X and Y dimensions from `-90` to `90`.
+    * These properties are `0` when the pen is perfectly perpendicular to the screen, and either `90` or `-90` when the pen is flat against the screen.
+  * `twist`
+    * This property reports the clockwise rotation of the pen around its axis in degrees.
+
+Use your mouse, touchscreen, or pen in this example to see how the properties represent the pointer.
```

Comment:
Again more explanation on what's going on in the sample to the right and what to take notice of